connector/connector.js

```diff
@@ -70,3 +70,35 @@ connector.createMiddleware = function(phase, data, cb) {
   middlewareDef,
   cb);
 };
+
+connector.createModel = function(id, data, cb) {
```

Comment:
I see, why don't we rename this to workspace-connector? It looks like you're basically extending connector (well mixing in new behaviours).